src/bun.js/test/jest.zig

```diff
@@ -1132,15 +1153,6 @@ pub const DescribeScope = struct {
         var i: TestRunner.Test.ID = 0;
 
         if (this.shouldEvaluateScope()) {
-            if (this.runCallback(globalObject, .beforeAll)) |err| {
```

Comment:
This is not needed anymore, since the new `runBeforeAll` function will go up the parent chain to run all `beforeAll` hooks